spa/src/utilities/isPortalAppPath.ts

```diff
@@ -1,7 +1,8 @@
 import { PORTAL } from 'routes';
 
 function isPortalAppPath() {
-  return window.location.pathname.indexOf(PORTAL.ENTRY) === 0;
+  // Check if current path has at least "/portal" at the start of the path
+  return window.location.pathname.indexOf(PORTAL.ENTRY_NO_SLASH) === 0;
```

Comment:
```suggestion
  return window.location.pathname.startsWith(PORTAL.ENTRY.replace(/\/$/, ''));
```
How would you feel about this? It would mean you wouldn't have to change anything in routes.ts.

Reply:
I like that idea. RegEx is not my strong point 😅 
[38035a9](https://github.com/newsrevenuehub/rev-engine/pull/1605/commits/38035a919283c7d2514c226ca458353e73129832)